src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-common.sch

```diff
@@ -57,6 +57,33 @@
     </xsl:for-each>
   </xsl:function>
 
+  <xsl:key name="groupKey"
```

Comment:
I don't think this is required anymore as I don't see any usage of `groupKey`.
Originally it was used however later I switched to xsl:for-each-group and it does it's own grouping.
Try removing this to see if it still works.

Reply:
No, you cannot remove this or the language code will be missing

![image](https://github.com/metadata101/iso19139.ca.HNAP/assets/74916635/48ccce8d-041b-4ebb-ad7f-32d0332b8638)

Reply:
ok. I tried and there is no ill effect. This groupKey can be removed.